static/src/javascripts/projects/common/modules/spacefinder.ts

```diff
@@ -87,6 +87,24 @@ const defaultOptions: SpacefinderOptions = {
 	debug: false,
 };
 
+const debugMinAbove = (body: HTMLElement, minAbove: number): void => {
```

Comment:
I feel like this should live in the `mark-candidates` file, as it's a part of debugging spacefinder, but then again the minAbove red line isn't a _candidate_ so maybe shouldn't live there.

@simonbyford Would it make sense to rename the `mark-candidates` file to `spacefinder-debug-tool`? Then we could have everything related to debugging spacefinder in the same place.

Reply:
oh yeah, I like this idea